go.mod

```diff
@@ -1,6 +1,6 @@
 module dp-frontend-cookie-controller
 
-go 1.13.7
+go 1.13.8
```

Comment:
Should just be `1.13`
Module files should not contain patch versions

Reply:
Jess is right